gc/verbose/handler_standard/VerboseHandlerOutputStandard.cpp

```diff
@@ -404,10 +426,11 @@ MM_VerboseHandlerOutputStandard::handleConcurrentMarkEnd(J9HookInterface** hook,
 	uint64_t duration = 0;
 	bool deltaTimeSuccess = getTimeDeltaInMicroSeconds(&duration, stats->_startTime, stats->_endTime);
 	const char* cardCleaningReasonString = getCardCleaningReasonString(collectionStats->getCardCleaningReason());
+	const char* terminationReason = getConcurrentTerminationReason(stats);
 
-	handleGCOPOuterStanzaStart(env, "trace", stats->_cycleID, duration, deltaTimeSuccess, cardCleaningReasonString);
+	handleGCOPOuterStanzaStart(env, "trace", stats->_cycleID, duration, deltaTimeSuccess, terminationReason);
```

Comment:
We don't have anything that records termination reason for CM (`getConcurrentTerminationReason` only works for CS), so this will always be NULL and never print. You can just remove the `terminationReason` param from this method.

Reply:
Top level `<concurrent-end>` stanza (MM_VerboseHandlerOutput::handleConcurrentEnd) reports termination reason. Again this is always null for CM but works for CS, I think after these changes are in, we should add a termination mechanism to CM. This can be added as part of these changes, but let's not delay this further so we'll do it separately